ogc-rs/src/constants.rs

```diff
@@ -0,0 +1,674 @@
+pub const GX_PERF0_XF_BOT_CLKS: u8 = 7;
+pub const GX_PERF0_XF_REGLD_CLKS: u8 = 8;
+pub const GX_PERF0_XF_REGRD_CLKS: u8 = 9;
+pub const GX_PERF0_CLIP_RATIO: u8 = 10;
+pub const GX_PERF0_TRIANGLES: u8 = 11;
+pub const GX_PERF0_TRIANGLES_CULLED: u8 = 12;
+pub const GX_PERF0_TRIANGLES_PASSED: u8 = 13;
+pub const GX_PERF0_TRIANGLES_SCISSORED: u8 = 14;
+pub const GX_PERF0_TRIANGLES_0TEX: u8 = 15;
+pub const GX_PERF0_TRIANGLES_1TEX: u8 = 16;
+pub const GX_PERF0_TRIANGLES_2TEX: u8 = 17;
+pub const GX_PERF0_TRIANGLES_3TEX: u8 = 18;
+pub const GX_PERF0_TRIANGLES_4TEX: u8 = 19;
+pub const GX_PERF0_TRIANGLES_5TEX: u8 = 20;
+pub const GX_PERF0_TRIANGLES_6TEX: u8 = 21;
+pub const GX_PERF0_TRIANGLES_7TEX: u8 = 22;
+pub const GX_PERF0_TRIANGLES_8TEX: u8 = 23;
+pub const GX_PERF0_TRIANGLES_0CLR: u8 = 24;
+pub const GX_PERF0_TRIANGLES_1CLR: u8 = 25;
+pub const GX_PERF0_TRIANGLES_2CLR: u8 = 26;
+pub const GX_PERF0_QUAD_0CVG: u8 = 27;
+pub const GX_PERF0_QUAD_NON0CVG: u8 = 28;
+pub const GX_PERF0_QUAD_1CVG: u8 = 29;
+pub const GX_PERF0_QUAD_2CVG: u8 = 30;
+pub const GX_PERF0_QUAD_3CVG: u8 = 31;
+pub const GX_PERF0_QUAD_4CVG: u8 = 32;
+pub const GX_PERF0_AVG_QUAD_CNT: u8 = 33;
+pub const GX_PERF0_CLOCKS: u8 = 34;
+pub const GX_PERF0_NONE: u8 = 35;
+pub const GX_PERF1_TEXELS: u8 = 0;
+pub const GX_PERF1_TX_IDLE: u8 = 1;
+pub const GX_PERF1_TX_REGS: u8 = 2;
+pub const GX_PERF1_TX_MEMSTALL: u8 = 3;
+pub const GX_PERF1_TC_CHECK1_2: u8 = 4;
+pub const GX_PERF1_TC_CHECK3_4: u8 = 5;
+pub const GX_PERF1_TC_CHECK5_6: u8 = 6;
+pub const GX_PERF1_TC_CHECK7_8: u8 = 7;
+pub const GX_PERF1_TC_MISS: u8 = 8;
+pub const GX_PERF1_VC_ELEMQ_FULL: u8 = 9;
+pub const GX_PERF1_VC_MISSQ_FULL: u8 = 10;
+pub const GX_PERF1_VC_MEMREQ_FULL: u8 = 11;
+pub const GX_PERF1_VC_STATUS7: u8 = 12;
+pub const GX_PERF1_VC_MISSREP_FULL: u8 = 13;
+pub const GX_PERF1_VC_STREAMBUF_LOW: u8 = 14;
+pub const GX_PERF1_VC_ALL_STALLS: u8 = 15;
+pub const GX_PERF1_VERTICES: u8 = 16;
+pub const GX_PERF1_FIFO_REQ: u8 = 17;
+pub const GX_PERF1_CALL_REQ: u8 = 18;
+pub const GX_PERF1_VC_MISS_REQ: u8 = 19;
+pub const GX_PERF1_CP_ALL_REQ: u8 = 20;
+pub const GX_PERF1_CLOCKS: u8 = 21;
+pub const GX_PERF1_NONE: u8 = 22;
+pub const GX_TLUT0: u8 = 0;
+pub const GX_TLUT1: u8 = 1;
+pub const GX_TLUT2: u8 = 2;
+pub const GX_TLUT3: u8 = 3;
+pub const GX_TLUT4: u8 = 4;
+pub const GX_TLUT5: u8 = 5;
+pub const GX_TLUT6: u8 = 6;
+pub const GX_TLUT7: u8 = 7;
+pub const GX_TLUT8: u8 = 8;
+pub const GX_TLUT9: u8 = 9;
+pub const GX_TLUT10: u8 = 10;
+pub const GX_TLUT11: u8 = 11;
+pub const GX_TLUT12: u8 = 12;
+pub const GX_TLUT13: u8 = 13;
+pub const GX_TLUT14: u8 = 14;
+pub const GX_TLUT15: u8 = 15;
+pub const GX_BIGTLUT0: u8 = 16;
+pub const GX_BIGTLUT1: u8 = 17;
+pub const GX_BIGTLUT2: u8 = 18;
+pub const GX_BIGTLUT3: u8 = 19;
+pub const GX_MAX_VTXDESC: u8 = 26;
+pub const GX_MAX_VTXDESC_LISTSIZE: u8 = 27;
+pub const GX_MAX_VTXATTRFMT: u8 = 26;
+pub const GX_MAX_VTXATTRFMT_LISTSIZE: u8 = 27;
+pub const GX_MAX_Z24: u32 = 16777215;
```

Comment:
Are these constants able to be referenced from ``ogc-sys``?

Reply:
Yes. I've moved `constants.rs` to `ogc-sys/src/constants.rs` and added them inside `lib.rs`.